Zend/tests/generators/bug78434.phpt

```diff
@@ -0,0 +1,28 @@
+--TEST--
+Bug #78434: Generator yields no items after valid() call
```

Comment:
I was a bit wrong in the bug description. The generator just skips the first item. See https://3v4l.org/2HBSX

```suggestion
Bug #78434: Generator skips first item after valid() call
```